index.html

```diff
@@ -184,5 +264,102 @@ <h4 class="top-banner__subtitle">
         </div>
       </div>
     </section>
+    <section class="why-us">
+      <div class="why-us__container">
+        <div class="why-us__main-content">
+          <h2 class="why-us__title">WHY CHOOSE US</h2>
+          <p class="why-us__subtitle">
+            Problems trying to resolve the conflict between the two major realms
+            of Classical physics: Newtonian mechanics
+          </p>
+        </div>
+        <div class="why-us__grid-container">
+          <div class="grid-item">
+            <div class="grid-item__heading">
+              <img
+                src="img-why-us/case.svg"
+                class="grid-item__img"
+                alt="case image"
+              />
+              <h5>Digital Marketing</h5>
+            </div>
+            <p class="grid-item__paragraph">
+              the quick fox jumps over the lazy dog the quick fox jumps over the
+              lazy dog
+            </p>
+          </div>
+          <div class="grid-item">
+            <div class="grid-item__heading">
+              <img
+                src="img-why-us/shop.svg"
+                class="grid-item__img"
+                alt="shop image"
+              />
+              <h5>Digital Marketing</h5>
+            </div>
+            <p class="grid-item__paragraph">
+              the quick fox jumps over the lazy dog the quick fox jumps over the
+              lazy dog
+            </p>
+          </div>
+          <div class="grid-item">
+            <div class="grid-item__heading">
+              <img
+                src="img-why-us/notebook.svg"
+                class="grid-item__img"
+                alt="notebook image"
+              />
+              <h5>Business Growing</h5>
+            </div>
+            <p class="grid-item__paragraph">
+              the quick fox jumps over the lazy dog the quick fox jumps over the
+              lazy dog
+            </p>
+          </div>
+          <div class="grid-item">
+            <div class="grid-item__heading">
+              <img
+                src="img-why-us/users.svg"
+                class="grid-item__img"
+                alt="users image"
+              />
+              <h5>National top 50 firms</h5>
+            </div>
+            <p class="grid-item__paragraph">
+              the quick fox jumps over the lazy dog the quick fox jumps over the
+              lazy dog
+            </p>
+          </div>
+          <div class="grid-item">
+            <div class="grid-item__heading">
+              <img
+                src="img-why-us/stats.svg"
```

Comment:
good for using svg's for icons

Reply:
After you said "don't get scared", I got scared :D
Thanks for the comprehensive feedback! Will try to apply all the needed changes.